client/src/rr/dnssec/keypair.rs

```diff
@@ -241,11 +242,12 @@ impl KeyPair {
     /// # Return
     ///
     /// the KEY record data
+    #[allow(deprecated)]
     pub fn to_sig0key(&self, algorithm: Algorithm) -> DnsSecResult<KEY> {
         self.to_public_bytes()
             .map(|bytes| {
                      KEY::new(Default::default(),
-                              Default::default(),
+                              KeyUsage::Zone,
```

Comment:
IMHO there is no good default, as both `KeyUsage::Entity` and `KeyUsage::Host` are equally valid options, as far as I can tell.
But: Changing the signature of `fn to_sig0key` would be an API breaking change, so adding `fn to_sig0key_with_usage` may still be the better option.